cmd/manager/manager.go

```diff
@@ -61,9 +66,19 @@ var (
 	configPath           string
 	enableLeaderElection bool
 	featureGates         string
-	version              string
+	operatorVersion      string
 )
 
+func requireKubernetesVersion(version *version.Info, major, minor int) {
```

Comment:
Maybe rename these variables into `requiredVersion` and `receivedVersion` or something like that? I was a bit confused which is which

Reply:
Changed to `serverVersion`, `minMajor` and `minMinor`.